lib/housekeeping.js

```diff
@@ -1,5 +1,6 @@
 const assert = require('assert');
 const {runAWSRequest} = require('./aws-request');
+const util = require('util');
```

Comment:
I didn't know about this module!  However, since the final code doesn't use it, we should remove the `require` as well.

Reply:
Makes sense. 😄